packages/wrangler/src/config/validation.ts

```diff
@@ -218,7 +218,7 @@ export function normalizeAndValidateConfig(
 			activeEnv = normalizeAndValidateEnvironment(
```

Comment:
Given they're now the same, we should move the `activeEnv` calculation outside of the Pages/Workers gate